meinberlin/apps/plans/assets/plans_map.jsx

```diff
@@ -321,10 +325,38 @@ class PlansMap extends React.Component {
               </ul>
             </div>
             &nbsp;
-            <select onChange={this.onParticipationFilterChange.bind(this)} className="u-inline btn btn--light">
-              <option value="-1">{django.gettext('Participation')}: {django.gettext('All')}</option>
-              <option value="1">{django.gettext('Participation')}: {django.gettext('Planned')}</option>
-            </select>
+            <div className="dropdown ">
+              <button type="button" className="dropdown-toggle btn btn--light btn--select" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" id="id_filter_participation">
+                {django.gettext('Participation')}: {participationNames[this.state.filters.participation] || django.gettext('All')}
+                <i className="fa fa-caret-down" aria-hidden="true" />
+              </button>
+              <ul aria-labelledby="id_filter_participation" className="dropdown-menu">
+                <li>
+                  <button
+                    type="button"
+                    className="dropdown-item select-item"
+                    value="-1"
+                    onClick={this.onParticipationFilterChange.bind(this)}>
+                    {django.gettext('All')}
+                  </button>
```

Comment:
Is there a specific reason why you used the `select-item` class here? There is no icon, so I see no real reason.

Reply:
oh this has been probably a copy and paste error. will fix it.